oktaawscli/aws_auth.py

```diff
@@ -149,10 +149,25 @@ def __extract_available_roles_from(assertion):
         return roles
 
     @staticmethod
-    def __create_options_from(roles):
+    def __create_options_from(roles, assertion, lookup=False):
         options = []
         for index, role in enumerate(roles):
-            options.append("%d: %s" % (index + 1, role.role_arn))
+            if lookup:
+                creds = AwsAuth.get_sts_token(role.role_arn, role.principal_arn, assertion, duration=900)
+                access_key_id = creds['AccessKeyId']
+                secret_access_key = creds['SecretAccessKey']
+                session_token = creds['SessionToken']
+                client = boto3.client('iam',
```

Comment:
`arn_region = role.principal_arn.split(':')[1]
            if arn_region == 'aws-us-gov':
                iam_region = 'us-gov-west-1'`
without this, it will fail if us gov aws account is being used

Reply:
Fixed.